config/config.go

```diff
@@ -58,6 +59,21 @@ var (
 	_devEnv = "development"
 )
 
+// AppRoot returns the root directory of your application. UberFx developers
+// can edit this via the APP_ROOT environment variable. If the environment
+// variable is not set then it will fallback to the current working directory.
+// This is often used for resolving relative paths in your service.
+func AppRoot() string {
+	if appRoot := os.Getenv(_appRoot); appRoot != "" {
+		return appRoot
+	}
+	if cwd, err := os.Getwd(); err != nil {
+		panic(fmt.Sprintf("Unable to get the current working directory: %s", err.Error()))
+	} else {
```

Comment:
i need it for the scope, but it is a little awkward. i could split the assignment and the error check.